server/api/utils/projects/leader.py

```diff
@@ -81,6 +81,7 @@ def store_project(
     ) -> tuple[typing.Optional[mlrun.common.schemas.Project], bool]:
         self._enrich_project(project)
         mlrun.projects.ProjectMetadata.validate_project_name(name)
+        mlrun.projects.ProjectMetadata.validate_project_labels(project.metadata.labels)
```

Comment:
why not use the same common `_validate_project` (put it in member instead of follower)